lighthouse-cli/run.js

```diff
@@ -64,25 +64,25 @@ function getDebuggableChrome(flags) {
 
 /** @return {never} */
 function showConnectionError() {
-  console.error('Unable to connect to Chrome');
+  log.error('CLI runtime', 'Unable to connect to Chrome');
```

Comment:
I think we can just have these be `'CLI'`, that matches our two other log statements from the CLI 👍

Reply:
Done